docs/index.md

```diff
@@ -408,35 +412,58 @@ problems by predicting one field of the dataset, the objective field,
 based on the values of the other fields, the input fields. The prediction
 is made using a logistic function whose argument is a linear combination
 of the predictor's values. Check the
-[developers documentation](https://ozone.dev.bigml.com/developers/logisticregressions)
+[developers documentation](https://ozone.dev.bigml.com/api/logisticregressions)
 for a detailed description. These resources
 are handled through `bigml.LogisticRegression`.
 
 - **associations** These resources are models to discover the existing
 associations between the field values in your dataset. Check the
-[developers documentation](https://ozone.dev.bigml.com/developers/associations)
+[developers documentation](https://ozone.dev.bigml.com/api/associations)
 for a detailed description. These resources
 are handled through `bigml.Association`.
 
+- **topic models** These resources are models to discover topics underlying a
+collection of documents. Check the
+[developers documentation](https://ozone.dev.bigml.com/api/topicmodels)
+for a detailed description. These resources
+are handled through `bigml.TopicModel`.
+
+- **topic distributions** These resources contain the distribution of
+probabilites
```

Comment:
typo: probabilities.  also, i'm not sure i understand the sentence... maybe "the probabilities for a document to belong"